controllers/script.js

```diff
@@ -246,11 +246,14 @@ var getScriptPageTasks = function (aOptions) {
 
   // Setup the flagging UI
   tasks.push(function (aCallback) {
-    // Can't flag when not logged in or when user owns the script.
-    if (!authedUser || aOptions.isOwner) {
-      aCallback();
-      return;
-    }
+    // WARNING: Possible duplicated code from flagLib.flaggable
+
+      // Can't flag when not logged in or when user owns the script.
+      if (!authedUser || aOptions.isOwner) {
```

Comment:
Tested actual on dev and behaves identical with removal... found another duplicate in `remove.js`... testing.